### PR TITLE
add amount field to faucet

### PR DESCRIPTION
**Pull request by jonathanlei**
## High Level Overview of Change

related to DGE [649](https://ripplelabs.atlassian.net/jira/software/projects/DGE/boards/524?selectedIssue=DGE-649)

### Context of Change
Add amount field to allow custom amount to be passed in for faucet transactions. 

### Type of Change
- [X ] New feature (non-breaking change which adds functionality)

## Test Plan
changed integration test, unit tests and integration tests passed.

### CODE REVIEW

File: packages/xrpl/src/Wallet/fundWallet.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ const MAX_ATTEMPTS = 20
  * ```
  *
  * @param this - Client.
+ * @param amount - A custom amount to fund, if undefined or null, the default amount will be 1000.
```
Comment (JST5000) [inline packages/xrpl/src/Wallet/fundWallet.ts:43]: I think amount should be added to the `options` object as an optional parameter. (Currently this would actually be a breaking change since it changes the order of parameters, so someone who had code like
`client.fundWallet(wallet)` would not longer match the code signature)

Comment (jonathanlei) [reply packages/xrpl/src/Wallet/fundWallet.ts]: changed

**Review by ckniffen — APPROVED** [review]

File: packages/xrpl/src/Wallet/fundWallet.ts [hunk 2]
```diff
@@ -76,6 +79,7 @@ async function fundWallet(
     new TextEncoder().encode(
       JSON.stringify({
         destination: walletToFund.classicAddress,
+        xrpAmount: options?.amount,
```
Comment (JST5000) [inline packages/xrpl/src/Wallet/fundWallet.ts:82]: What happens if the options pass in an invalid amount? (One that goes beyond how much the Testnet should be able to fund?) - Probably don't need to make it a test, but curious about the edge case inputs.

Comment (jonathanlei) [reply packages/xrpl/src/Wallet/fundWallet.ts:82]: faucet repo has checks on the amounts - if input is invalid it will return an error
https://github.com/xpring-eng/testnet-faucet/blob/336ad485a51fbf63a9d5c335979c68a9886af563/index.js#L135

**Review by JST5000 — APPROVED** [review]
LGTM

**Comment by JST5000** [comment]
Oh, can you update the HISTORY.md file in the xrpl package?

**Comment by jonathanlei** [comment]
HISTORY.md updated now
